app/models/patient.rb

```diff
@@ -44,6 +44,8 @@ class Patient < ApplicationRecord
   has_many :appointments
   has_many :notifications
   has_many :treatment_group_memberships, class_name: "Experimentation::TreatmentGroupMembership"
+  has_many :treatment_groups, through: :treatment_group_memberships, class_name: "Experimentation::TreatmentGroup"
```

Comment:
This is not related to the ticket. It's something I found very helpful while doing something else, so I decided to add it.